### ISSUE TITLE
Cannot get parent or parentID of a channel.

### ISSUE THREAD

**Issue by notlouis** [discordjs/discord.js#4918]
**Please describe the problem you are having in as much detail as possible:**
Cannot seem to get the parent or parentID of a channel with a parent.

**Include a reproducible code sample here, if possible:**

```js
console.log(message.channel.parent) //returns null
console.log(message.channel.parentID) //returns null
```

**Further details:**

- discord.js version: 12.4.0
- Node.js version: 14.14.0
- Operating system: Lubuntu
- Priority this issue should have – please be realistic and elaborate if possible:
Medium - High, lots of commands rely on categories and being unable to get them can be a big issue.

**Relevant client options:**

- partials: none
- gateway intents: none
- other: none

- [x] I have also tested the issue on latest master, commit hash:

**Comment by Fyrlex** [comment]
if there is no category, this will obviously happen i believe. you should make a catch to prevent errors.

**Comment by almostSouji** [comment]
CR on `12.4.0`, though discord seems to be having hiccups at the moment I do not believe that to be related

**Comment by notlouis** [comment]
> if there is no category, this will obviously happen i believe. you should make a catch to prevent errors.

There is a category, i've tried it on every channel in the server and with the same result. I'll try fetching shortly.

**Comment by markito12** [comment]
same here

**Comment by adrienbrignon** [comment]
Hello,

I don't know if Discord is having issues at the moment but I am also unable to retrieve the parent category of my channels.

![image](https://user-images.githubusercontent.com/10183277/97615593-cc5cb900-1a1b-11eb-8476-98e6bca04570.png)
The parent of my voice channel 'debug' is `null`.

**Comment by SpaceEEC** [comment]
That was fixed with `12.4.1`, you may want to update your discord.js version.
